### ISSUE TITLE
Problem with LESS &:extend()

### ISSUE THREAD

**Issue by zpiman** [Glavin001/atom-beautify#159]
Hi, I found a problem when beautifying *LESS* files
When I have something like this:
```less
.some_class {
   &:extend(.some_other_class);
  //... some more code
}
```
*atom-beautify* adds a space between the `&:` and the `extend` so the result looks like this: 
```less
.some_class {
   &: extend(.some_other_class);
  //... some more code
}
```
The resulting code does not compile :disappointed: 
This does not happen for `&:hover`
My best guess is that the `extend` keyword is not on some list of verbs whereas `hover` is. I set atom to beautify my code on save, now I had to disable it because of this problem.
Thank you in advance!

**Comment by Glavin001** [comment]
Published to v0.18.0. Happy New Year! :smiley:

**Comment by zpiman** [comment]
Happy New Year to you too! And thank you! Now I can enable *beautify* again :smile:
